### ISSUE TITLE
Error while running KSCN node with subbridge enabled

### ISSUE THREAD

**Issue by jyeoniii** [klaytn/klaytn#1447]
**Describe the bug**
KSCN process was unexpectedly down with the error log below. 
Subbridge and anchoring was enabled, and no tx was sent to the chain when an incident occurred.

**How to reproduce**
Cannot reproduce. When starting the node again, it works as normal. 

**Expected behavior**
Node works as normal without any error. 

**Attachments**
If applicable, add data, logs or screenshots to help explain your problem.
```
INFO[06/20,04:40:49 Z] [45] Contract Event Loop Running Stop by receivedSub.Err()  err=nil
INFO[06/20,04:40:49 Z] [45] Contract Event Loop Running Stop by receivedSub.Err()  err=nil
fatal error: concurrent map read and map write

goroutine 184 [running]:
runtime.throw(0x1d2b542, 0x21)
        /usr/local/go/src/runtime/panic.go:1116 +0x72 fp=0xc000f91c80 sp=0xc000f91c50 pc=0x44b5b2
runtime.mapaccess1(0x1a4b940, 0xc0006c1530, 0xc000f91cf4, 0x30a7620)
        /usr/local/go/src/runtime/map.go:411 +0x24f fp=0xc000f91cc0 sp=0xc000f91c80 pc=0x42318f
github.com/klaytn/klaytn/node/sc.(*BridgeManager).UnsubscribeEvent(0xc00003f1e0, 0xe6153c2bafa2399c, 0xe31a65a79161de38, 0x7fb3c132126c)
        /ext-go/1/src/github.com/klaytn/klaytn/node/sc/bridge_manager.go:984 +0xae fp=0xc000f91d18 sp=0xc000f91cc0 pc=0x16ea74e
github.com/klaytn/klaytn/node/sc.(*BridgeManager).RestoreBridges(0xc00003f1e0, 0xc000f91f40, 0x2)
        /ext-go/1/src/github.com/klaytn/klaytn/node/sc/bridge_manager.go:690 +0x365 fp=0xc000f91f00 sp=0xc000f91d18 pc=0x16e7165
github.com/klaytn/klaytn/node/sc.(*SubBridge).restoreBridgeLoop(0xc0006a4b60)
        /ext-go/1/src/github.com/klaytn/klaytn/node/sc/subbridge.go:594 +0x1ab fp=0xc000f91fd8 sp=0xc000f91f00 pc=0x1705b0b
runtime.goexit()
        /usr/local/go/src/runtime/asm_amd64.s:1374 +0x1 fp=0xc000f91fe0 sp=0xc000f91fd8 pc=0x4844e1
created by github.com/klaytn/klaytn/node/sc.(*SubBridge).SetComponents
        /ext-go/1/src/github.com/klaytn/klaytn/node/sc/subbridge.go:379 +0x3bc

goroutine 1 [chan receive, 159 minutes]:
github.com/klaytn/klaytn/node.(*Node).Wait(0xc0003d4600)
        /ext-go/1/src/github.com/klaytn/klaytn/node/node.go:635 +0x7c
github.com/klaytn/klaytn/cmd/utils/nodecmd.RunKlaytnNode(0xc00023c580, 0x0, 0x0)
        /ext-go/1/src/github.com/klaytn/klaytn/cmd/utils/nodecmd/defaultcmd.go:53 +0x53
gopkg.in/urfave/cli%2ev1.HandleAction(0x19ce140, 0x1da1fd0, 0xc00023c580, 0xc000392d80, 0x0)
        /go/pkg/mod/gopkg.in/urfave/cli.v1@v1.20.0/app.go:490 +0x82
gopkg.in/urfave/cli%2ev1.(*App).Run(0xc00011e680, 0xc000100800, 0x3f, 0x40, 0x0, 0x0)
        /go/pkg/mod/gopkg.in/urfave/cli.v1@v1.20.0/app.go:264 +0x5f5
main.main()
        /ext-go/1/src/github.com/klaytn/klaytn/cmd/kscn/main.go:99 +0xa5
```

**Environment (please complete the following information)**
 - Klaytn version or git revision that exhibits the issue `$ ken version`
 - go version `$ go version`
 - OS and its version [e.g. macOS/10.14]

* Klaytn version: Klaytn v1.8.2 (Installed via `kscn-v1.8.2-0-linux-amd64.tar.gz` in [Download](https://ko.docs.klaytn.foundation/node/download/v1.8.2) page.
* OS: Amazon Linux 2 Kernel 5.10 AMI 2.0.20220426.0 x86_64 HVM gp2

**Additional context**

**Comment by hyunsooda** [comment]
Thanks for the report and detailed log. The crash seemed to happen from concurrent access to [map variable](https://github.com/klaytn/klaytn/blob/v1.8.2/node/sc/bridge_manager.go#L984), which is currently not aware of concurrent access (i.e., no mutex guard).  #1438 has commits that fix the concurrency access and has not merged yet.

**Comment by aidan-kwon** [comment]
Resolved via #1438
